sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -2129,4 +2129,17 @@ public void testSelectPartitionValueFromInformationSchema() throws Exception {
         execute("select values['p'], values['obj[''p'']'] from information_schema.table_partitions");
         assertThat(printedTable(response.rows()), is("1| 10\n"));
     }
+
+    @Test
+    public void testRefreshIgnoresClosedPartitions() {
+        execute("create table t (x int, p int) " +
+                "partitioned by (p) clustered into 1 shards with (number_of_replicas = 0, refresh_interval = 0)");
+        execute("insert into t (x, p) values (1, 1), (2, 2)");
+        execute("alter table t partition (p = 2) close");
+        execute("refresh table t");
+        assertThat(
```

Comment:
shouldn't we check the row count of refresh? (`@UseJdbc(0)`) I guess.

Reply:
+1